recipes/noiseflow/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ae0f060010667a924f7281976762d85cf5ba2402ccb404a0bab7a5fc893188fe
+
+build:
+  number: 0
+  skip: True  # [python_impl == 'pypy']
+  script: 
+    - build_osx.sh  # [osx]
+    - build_linux.sh  # [linux]
+    - build_win.bat  # [win]
+  script_env: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+
+requirements:
+  build:
+    - "{{ compiler('cxx') }}"
+    - python                              
+    - poetry-core
+    - setuptools                          
+    - cmake
+    - numpy
+    - pip  
+    - make  # [unix]
+    - ninja  
+
+  host:
+    - pip
+    - numpy >=1.25.0
+    - fftw  # [unix]
```

Comment:
I'm surprised that you have `# [unix[` for key dependencies. Why is that?

Reply:
I just build the c++ lib for macos and linux, not for windows, cuz I can't compile it

Reply:
Do you mean: I will never be able to compile on Windows? Then you should skip it

Reply:
To have this dependencies on `unix` but enable CPP only in `linux` (above) seems a bit contradictory. Could you clarify ?

Reply:
The `fftw` is not used in win platform in my package. The c++ extension is only used in macos and linux.

Reply:
I'm sorry, what do you mean `but enable CPP only in linux (above)`?